src/clj/chronograph/handlers/timer.clj

```diff
@@ -0,0 +1,138 @@
+(ns chronograph.handlers.timer
+  (:require [chronograph.domain.timer :as timer]
+            [next.jdbc :as jdbc]
+            [chronograph.db.core :as db]
+            [chronograph.domain.task :as task]
+            [ring.util.response :as response]
+            [clojure.spec.alpha :as s]))
+
+(defn create
+  "Authorized users may create a timer for the give task id."
+  [{{:keys [task-id note] :as body} :body
+    {user-id :users/id} :user
+    :as _request}]
+  (jdbc/with-transaction [tx db/datasource]
+    (cond
+      (not (s/valid? :handlers.timer/create-request-body body))
+      (response/bad-request
+       {:error "Invalid Task ID or Note."})
+
+      (empty? (task/find-by-id tx task-id))
```

Comment:
We should check that the user is a member of the organization that the task is under. If not, it should reject the request with a 403 Forbidden.

Reply:
Done.